frontend/components/Item/CollectionTabs.vue

```diff
@@ -41,23 +42,22 @@ export default Vue.extend({
     };
   },
   computed: {
-    ...mapGetters('items', ['getBoxsetChildren']),
+    ...mapStores(itemsStore),
     children(): BaseItemDto[] {
-      return this.getBoxsetChildren(this.item.Id);
+      return this.items.getChildrenOfParent(this.item.Id) || [];
```

Comment:
I'm afraid this is not enough, as the component tabs elements based on type.

In order not to have regressions, it would be necessary to reintroduce the logic that I had inserted at the time.

```suggestion
     children(): ItemsByType[] {
      const list = this.items.getChildrenOfParent(this.item.Id);

      if (list) {
        const c: { [key: string]: BaseItemDto[] } = {};

        list.forEach((item: BaseItemDto) => {
          if (item.Type && typeof item.Type === 'string') {
            if (!c[item.Type]) c[item.Type] = [];

            c[item.Type].push(item);
          }
        });

        const perTypeList: ItemsByType[] = Object.keys(c).map((type) => {
          return { Type: type, Items: c[type] };
        });

        return perTypeList;
      } else return [];
    }
```
It needs this, declared somewhere

```typescript
interface ItemsByType {
  Type: string;
  Items: BaseItemDto[];
}
```

Reply:
I fixed this with a [simpler logic](https://github.com/jellyfin/jellyfin-vue/pull/1746/commits/ebd85f3721f15137582a37066feb950005ce92d2#diff-4edbc1f6f9c1a1e8f29ee928046706a4d29a182a30b7309f00dbb3a3e5bcdbd6R46-R61)

The tabs still need proper translation support though

Reply:
I fixed this with a [much more easier logic](https://github.com/jellyfin/jellyfin-vue/blob/pinia/frontend/components/Item/CollectionTabs.vue#L43)

The tabs still need proper translation support though

Reply:
Well done!

I absolutely have to see what lodash offers, because I see rather comfortable methods around here! 😄